Changelog.md

```diff
@@ -12,6 +12,7 @@ Compiler Features:
  * Parser: Unary plus is no longer recognized as a unary operator in the AST and triggers an error at the parsing stage (rather than later during the analysis).
  * SMTChecker: Properties that are proved safe are now reported explicitly at the end of the analysis. By default, only the number of safe properties is shown. The CLI option ``--model-checker-show-proved-safe`` and the JSON option ``settings.modelChecker.showProvedSafe`` can be enabled to show the full list of safe properties.
  * SMTChecker: Group all messages about unsupported language features in a single warning. The CLI option ``--model-checker-show-unsupported`` and the JSON option ``settings.modelChecker.showUnsupported`` can be enabled to show the full list.
+ * Standard JSON Interface: Add support to import AST via Standard JSON.
```

Comment:
```suggestion
 * Standard JSON Interface: Add experimental support for importing ASTs via Standard JSON.
```

Maybe better to call it "experimental" in the sense that we don't have nice error-reporting on invalid ASTs.